.cursorignore

```diff
@@ -0,0 +1,4 @@
+# Add directories or file patterns to ignore during indexing (e.g. foo/ or *.csv)
```

Comment:
Whats this? Cursor? Graphana?

Reply:
Cursor is an AI IDE. this is a file that helps indexing things right,